### PR TITLE
Add new qmp test case "GUEST_PANICKED' event"

### PR DESCRIPTION
**Pull request by jingzhao84**
Add new case for checking "GUEST_PANICKED" event
ID: 1550843

Signed-off-by: jinzhao <jinzhao@redhat.com>

### CODE REVIEW

**Comment by jingzhao84** [comment]
(17/17) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_guest_panicked.from_guest: PASS (107.26 s)

**Comment by jingzhao84** [comment]
@yanan-fu @huangyum  could you help to review this pr?

Thanks
Jing

**Comment by CongLi** [comment]
please remove internal info.

**Comment by jingzhao84** [comment]
@CongLi  updated and thanks cong

File: qemu/tests/cfg/qmp_event_notification.cfg [hunk 1]
```diff
@@ -91,3 +91,8 @@
                 - qmp_none:
                     extra_params += " -watchdog i6300esb -watchdog-action none "
                     action_check = "none"
+        - qmp_guest_panicked:
+            no Windows
+            event_check = "GUEST_PANICKED"
+            event_cmd = service kdump stop;echo c >/proc/sysrq-trigger
```
Comment (hereischen) [inline qemu/tests/cfg/qmp_event_notification.cfg]: I think it should be`service kdump stop && echo c >/proc/sysrq-trigger`
If previous command failed with `;` the second one will run, which is not what we want. 
But with `&&` the second one will not run.

Comment (jingzhao84) [reply qemu/tests/cfg/qmp_event_notification.cfg]: @hereischen  updated and please help to review it? thanks

 (17/17) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_guest_panicked.from_guest: PASS (107.23 s)

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

File: qemu/tests/cfg/qmp_event_notification.cfg [hunk 2]
```diff
@@ -91,3 +91,8 @@
                 - qmp_none:
                     extra_params += " -watchdog i6300esb -watchdog-action none "
                     action_check = "none"
+        - qmp_guest_panicked:
```
Comment (yanan-fu) [inline qemu/tests/cfg/qmp_event_notification.cfg]: For pc and q35, pvpanic device is must for this case, so i suggest to add pvpanic device here even if this device is added by default in machine.cfg.  In this way , no dependence issue in the future.

Comment (jingzhao84) [reply qemu/tests/cfg/qmp_event_notification.cfg]: @yanan-fu updated and please review again?

 (17/18) smp_4.1024m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_guest_panicked.from_guest: PASS (107.09 s)

Thanks
Jing

File: qemu/tests/cfg/qmp_event_notification.cfg [hunk 3]
```diff
@@ -91,3 +91,10 @@
...
+            no Windows
+            enable_pvpanic = yes
+            ioport_pvpanic = 0x505
```
Comment (yanan-fu) [inline qemu/tests/cfg/qmp_event_notification.cfg]: only for q35 and pc,  no need for ppc,

Comment (jingzhao84) [reply qemu/tests/cfg/qmp_event_notification.cfg]: @yanan-fu update again and pls review it

(17/18) smp_4.1024m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_guest_panicked.from_guest.q35: PASS (107.44 s)

(17/18) smp_4.1024m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_guest_panicked.from_guest: PASS (107.23 s)

Thanks
Jing

**Comment by yanan-fu** [comment]
Ack-by: Yanan Fu <yfu@redhat.com>

**Comment by jingzhao84** [comment]
@luckyh  could you help to review this pr? thanks

File: qemu/tests/cfg/qmp_event_notification.cfg [hunk 4]
```diff
@@ -91,3 +91,11 @@
                 - qmp_none:
                     extra_params += " -watchdog i6300esb -watchdog-action none "
                     action_check = "none"
+        - qmp_guest_panicked:
+            no Windows
```
Comment (luckyh) [inline qemu/tests/cfg/qmp_event_notification.cfg]: Does windows not support this test?

File: qemu/tests/cfg/qmp_event_notification.cfg [hunk 5]
```diff
@@ -91,3 +91,11 @@
...
+            i440fx,q35:
+              enable_pvpanic = yes
+              ioport_pvpanic = 0x505
```
Comment (luckyh) [inline qemu/tests/cfg/qmp_event_notification.cfg]: Does the pvpanic device support all platform?

Comment (jingzhao84) [reply qemu/tests/cfg/qmp_event_notification.cfg]: @luckyh yes, but it didn't need to enable pvpanic for ppc arch

File: qemu/tests/cfg/qmp_event_notification.cfg (hunk 4 above)
Comment (jingzhao84) [reply qemu/tests/cfg/qmp_event_notification.cfg]: @luckyh  first, no care about windows and linux from qmp event part, second, the method of  windows is verify different with linux, also need mannual settings for windows

**Comment by jingzhao84** [comment]
@luckyh updated and could you help to review it again, thanks

**Pull request merged**